examples/tts/conf/de/fastpitch_align_44100.yaml

```diff
@@ -90,7 +90,10 @@ model:
       max_duration: 15  # change to null to include longer audios.
       min_duration: 0.1
       ignore_file: null
-      trim: false
+      trim: true
+      trim_top_db: 50,
```

Comment:
I am not sure what the rationale is for the default librosa config, other than it being very conservative. From my testing it seemed like trim_top_db = 30.0 worked well if you set trim_ref=np.max, and trim_top_db=50.0 if trim_ref=1.0.

Basically using trim_ref=np.max means it sets the threshold based on the maximum power in the audio sample, while using trim_ref=1.0 sets a constant threshold. So if trim_ref=np.max then the top_db needs to be smaller as it will lower the threshold based on how (loud?) the speaker is talking.

I am not sure what is best for a default, but we may as well make trim_ref a configurable value too if we want to use this.

Reply:
The default value for `trim_ref` is `np.max`, which actually take the max of amplitude of audio signal, instead of power spectrum (https://github.com/librosa/librosa/blob/f39ef0ed4931da25a35b47aa194f5d6b9605053c/librosa/effects.py#L439). For general purpose trimming, it would be better to access the dynamic reference, i.e. `np.max`. Choosing `trim_ref=1.0` requires the PCM normalized within [-1, 1] ahead I guess.

Reply:
This YAML config serves for general purpose, so it would better to enable trim option, but we should mention potential tweaks in the tutorial.

Reply:
I am ok with enabling the trimming by default. But before that, can we somehow confirm that trimming the audio for that particular dataset/tutorial does not negatively impact the final model?

Reply:
that's what I am doing now. Training with top_db=50. Note that this yaml is for German only.

Reply:
Default should be off so as not to hurt later projects with unexpected changes